shell/browser/native_window_views.cc

```diff
@@ -1318,6 +1342,51 @@ bool NativeWindowViews::IsVisibleOnAllWorkspaces() {
   return false;
 }
 
+void NativeWindowViews::SetVibrancy(const std::string& type) {
+#if defined(OS_WIN)
+  auto version = base::win::GetVersion();
+  if (version >= base::win::Version::WIN10) {
+    std::vector<uint8_t> bytes;
+    int gradientColor = 0;
+
+    // Acrylic blur is supported on Windows RS3 and above.
+    AccentState state = version >= base::win::Version::WIN10_RS3
+                            ? ACCENT_ENABLE_ACRYLICBLURBEHIND
+                            : ACCENT_ENABLE_BLURBEHIND;
+
+    // Disable vibrancy if an empty string is passed.
+    if (type.empty()) {
+      state = ACCENT_DISABLED;
+      // Allow setting a colored vibrancy from an #AARRGGBB hex string.
+    } else if (type[0] == '#') {
+      std::string color_string = type.substr(1);
+      if (!base::HexStringToBytes(color_string, &bytes))
+        return;
+      // Windows uses ABGR & HexStringToBytes returns ARBG so index accordingly.
+      gradientColor =
+          (bytes[0] << 24) | (bytes[3] << 16) | (bytes[2] << 8) | bytes[1];
+      // Allow forcing blur.
+    } else if (type == "blur") {
+      state = ACCENT_ENABLE_BLURBEHIND;
+    } else {
+      return;
+    }
+
+    AccentPolicy policy = {state, 2, gradientColor, 0};
+    WINCOMPATTRDATA data = {WCA_ACCENT_POLICY, &policy, sizeof(AccentPolicy)};
+
+    typedef BOOL(WINAPI * pSetWindowCompositionAttribute)(HWND,
+                                                          WINCOMPATTRDATA*);
+    const pSetWindowCompositionAttribute SetWindowCompositionAttribute =
+        (pSetWindowCompositionAttribute)GetProcAddress(
```

Comment:
use this pattern instead https://github.com/electron/electron/blob/c5ad7ed0cd05fe46aefedd0c3dbed580d6a5ce20/shell/browser/notifications/win/win32_desktop_notifications/common.h#L39-L42